rockcraft.yaml

```
@@ -23,6 +23,13 @@ services:
         group: mysql
 
 parts:
+    apt:
+      plugin: nil
+      override-build: |
+        apt update
+        apt -y upgrade
+        apt -y dist-upgrade
```

```suggestion
        apt -y dist-upgrade
```
does dist-upgrade do everything upgrade does?

---

From docs, it seems so - I need to check though.